python/ray/util/actor_group.py

```diff
@@ -96,7 +96,9 @@ def __init__(
         init_args: Optional[Tuple] = None,
         init_kwargs: Optional[Dict] = None,
     ):
-        ray._private.usage.usage_lib.record_library_usage("util.ActorGroup")
+        from ray._private.usage.usage_lib import record_library_usage
+
+        record_library_usage("util.ActorGroup")
```

Comment:
Is this change unrelated?

Reply:
It's related. It's an issue uncovered by the snippet test.

```
>>> ray._private.usage.usage_lib.record_library_usage("helo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'ray._private' has no attribute 'usage'
```